### ISSUE TITLE
First Run no SQL and errors on login

### ISSUE THREAD

**Issue by purplepixie** [purplepixie/freenats#27]
On setup - in welcome.php issue with firstrun flag:

Fatal error: Uncaught mysqli_sql_exception: Duplicate entry 'freenats.firstrun' for key 'PRIMARY' in /opt/freenats/server/base/nats.db.inc.php:66 Stack trace: #0 /opt/freenats/server/base/nats.db.inc.php(66): mysqli_query() #1 /opt/freenats/server/base/nats.cfg.inc.php(65): TNATS_DB->Query() #2 /opt/freenats/server/web/welcome.php(34): TNATS_Cfg->Set() #3 {main} thrown in /opt/freenats/server/base/nats.db.inc.php on line 66

Additionally reports that firstrun.php does not import the SQL files:

> and that firstrun.php does not import scheme.sql and default.sql

**Comment by purplepixie** [comment]
Issue is in nats.cfg.inc.php TNATS_Cfg::Set

````php
	function Set($var, $val, $perm = true)
	{
		global $NATS;
		$this->data[$var] = $val;
		if ($perm) {
			global $NATS;
			$q = "UPDATE fnconfig SET fnc_val=\"" . ss($val) . "\" WHERE fnc_var=\"" . ss($var) . "\"";
			$NATS->DB->Query($q);
			if ($NATS->DB->Affected_Rows() <= 0) // not already existing
			{
				$q = "INSERT INTO fnconfig(fnc_var,fnc_val) VALUES(\"" . ss($var) . "\",\"" . ss($val) . "\")";
				$NATS->DB->Query($q);
			}
		}
	}
````

Using UPDATE as a test won't work when the var and val are the same, needs a SELECT test.

**Comment by purplepixie** [comment]
Cannot replicate firstrun not running SQL - tested on fresh and upgrade schema

**Comment by purplepixie** [comment]
Set issue is fixed as of 1.30.9, however cannot replicate SQL as yet.

**Comment by purplepixie** [comment]
Issue that in PHP 8.1 (and presumably later) the MySQLi query in nats.cfg.inc.php on Load() would fail as exception not caught (and table doesn't exist). Added a silent exception handler.
